server/models/User.js

```diff
@@ -43,6 +43,9 @@ const User = sequelize.define(
       isSuspended() {
         return !!this.suspendedAt;
       },
+      isInvited() {
+        return !this.lastActiveAt;
+      },
```

Comment:
@tommoor I was able to get `isInvited` on the server by adding this getter – I _think_ this is okay pls lmk otherwise

Reply:
lgtm

Reply:
Doesn't look like this is needed anymore but I'm fine with leaving it